src/app/onion/learning-object-builder/components/content-upload/app/content-upload.module.ts

```diff
@@ -25,6 +25,7 @@ import { SharedModule } from '../../../../../shared/shared.module';
 import { ContextMenuModule } from 'ngx-contextmenu';
 import { UrlManagerComponent } from './upload/url-manager/url-manager.component';
 import { FileUploadStatusComponent } from './upload/file-upload-status/file-upload-status.component';
+import { TitleOfUrlComponent } from "./upload/url-manager/title-of-url/title-of-url.component";
```

Comment:
Can we potentially give this component a more descriptive name? TitleOfUrlComponent doesn't actually describe the component. Would UrlRowComponent be more descriptive? Since a url object has a title and a url property UrlRowComponent sounds a little less verbose and seems to be a better representation of the component.